setup.cfg

```diff
@@ -19,6 +19,8 @@ ignore =
     RST201,RST203,RST301,
     # Allow :nowrap: in math directives
     RST307
+    # Support LaTex in docstrings
+    D301
```

Comment:
This does not support LaTeX, it is to enable escaped backslashes in non-raw strings. This could be to enable line wrapping for long type names in docstrings that require escaped backslashes. However, I am not sure if the new Python updater couldn't be a raw string here.

Reply:
This has come up on many PRs where the long text strings needed to produce the desired formatting conflicts with the Google docstring requirement that certain descriptions fit on one line. To work around this, I suggested that we use `\` as a line continuation character which unfortunately cannot be used in a raw string. Therefore, to enable LaTeX in strings that need this line continuations, I added this ignore. This is discussed in detail in #1001.

If you would like to go and reformat all the existing documentation that needs this and that on all open PRs so we can remove the D301 ignore, feel free to. The alternative presented in #1001 is to apply a noqa to the entire string which is far too heavy handed in my opinion. It would allow any line in the docstring to run over the line length limit which would result in many future style debates in PRs and would make reviewing diffs of the documentation on small screens difficult.

Reply:
Fine, by me then. I would just like the comment to describe more of why this supports LaTeX in conjunction with line continuation.